internal/template-validator/webhooks/hook.go

```diff
@@ -154,11 +162,19 @@ func serve(resp http.ResponseWriter, req *http.Request, admit admitFunc) {
 		return
 	}
 
-	log.Log.V(8).Infof("admission review:\n%s", spew.Sdump(review))
+	if reviewJson, err := json.Marshal(review); err == nil {
+		log.Log.V(8).Infof("admission review:\n%s", string(reviewJson))
+	} else {
+		log.Log.V(8).Infof("admission review:\nCould not marshall review to json: %s", err.Error())
```

Comment:
Typo: marshal / marshall

Reply:
Can you dump out the review with `%v`?

Reply:
I think json is easier to read. `%v` is more verbose, I think it prints types of fields and addresses of pointers.